week-1/errors/3.js

```diff
@@ -1,7 +1,13 @@
 const cardNumber = 4533787178994213;
-const last4Digits = cardNumber.slice(-4);
+const last4Digits = String(cardNumber).slice(-4);
+console.log(last4Digits);
+
 
 // The last4Digits variable should store the last 4 digits of cardNumber
 // However, the code isn't working
 // Make and explain a prediction about why the code won't work
+
+/* The slice() method is a string method, which means it can only be used on string values, not numbers. 
```

Comment:
Great job on handling these JS tasks, @Ebtesammm!

Just a tiny one here: `slice` is also an array method.